codestodians/food-basket-seller
# [Feature: #154881426] Setup React.

#### What will this PR do?
- Setup the react application with all the necessary dependencies

#### Description of Task to be completed?
- Setup `webpack`.
- Update babel config
- Update eslint config
- Create react app
- Create app component
- Write test for sample component

#### How should this be manually tested?
- Check out into the `ft/create-react-app-154881426` branch.
- Run `yarn start` to start up the app.
- Run `yarn test` to run tests.

#### Any background context you want to provide?
- The seller repo is supposed to house creation of a store and addition of products. And so it becomes necessary to get it up and running so as to make possible the addition of stores and make progress with the application.

#### What are the relevant pivotal tracker stories?
- [#154881426](https://www.pivotaltracker.com/story/show/154881426)
